src/main.ts

```diff
@@ -64,26 +64,30 @@ async function run() {
       ? await github.listPullRequestFiles(client)
       : [];
 
-    core.info(`Fetched ${changedFiles.length} changed file(s)`);
+    if (onlyChanged) {
+      // Don't print "0 changed files" when we've disabled the option
+      core.info(`Fetched ${changedFiles.length} changed file(s)`);
+    }
```

Comment:
@stackptr this didn't seem to work...

The last run shows this log message, even though `only-changed: false`. Am I missing some JS nuance here?